modules/sql/managed-instance/main.bicep

```diff
@@ -340,7 +340,7 @@ module managedInstance_vulnerabilityAssessment 'vulnerability-assessment/main.bi
     recurringScansEmailSubscriptionAdmins: contains(vulnerabilityAssessmentsObj, 'recurringScansEmailSubscriptionAdmins') ? vulnerabilityAssessmentsObj.recurringScansEmailSubscriptionAdmins : false
     recurringScansIsEnabled: contains(vulnerabilityAssessmentsObj, 'recurringScansIsEnabled') ? vulnerabilityAssessmentsObj.recurringScansIsEnabled : false
     storageAccountResourceId: vulnerabilityAssessmentsObj.storageAccountResourceId
-    storageAccountFirewallOrVnet: contains(vulnerabilityAssessmentsObj, 'storageAccountFirewallOrVnet') ? vulnerabilityAssessmentsObj.storageAccountFirewallOrVnet : false
+    useStorageAccountAccessKey: contains(vulnerabilityAssessmentsObj, 'useStorageAccountAccessKey') ? vulnerabilityAssessmentsObj.useStorageAccountAccessKey : true
```

Comment:
I wonder if this should be false. Not using an access key but other means of authentication seem more secure, right?

Reply:
Also PSRule seems to be after [AAD Auth](https://github.com/lsnoddy/ResourceModules/actions/runs/6331186963/attempts/1#summary-17195207525). It sounds related, but I'm not sure

Reply:
I agree here. It probably makes better sense from a security standpoint to defuault to false.